CONTRIBUTING.md

```diff
@@ -85,25 +85,28 @@ development with the following:
 
   For Url:
 
-  `http://localhost:5000/`
+  `http://localhost:5000/auth/myusa/callback`
 
   For Redirect uri:
 
   `http://localhost:5000/auth/myusa/callback`
 
-2. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
+2. Under `Select the API Scopes that your Application will use`: select `Email
+   Address`.
+
+3. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
```

Comment:
I know that for myself when I first asked for an API key for the bot I had to have an 18F Slack admin create one for me. 

My guess is that people working on this will, unfortunately, need to create their own Slack org